examples/complete/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "public_key" {
+  description = "Public API key to authenticate to Atlas"
```

Comment:
Suggest to be more specific and say: "an Atlas cluster in your project", not just Atlas.